modules/distr-tracing-rn-fixed-issues.adoc

```diff
@@ -14,6 +14,8 @@ Fix - What did we change to fix the problem?
 Result - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
+* link:https://issues.redhat.com/browse/TRACING-2340[TRACING-2340] Fixed an issue where the jaeger-query UI login failed with the following error message: `oauthproxy.go:445: ErrorPage 500 Internal Error configmap "oauth-serving-cert" not found`. This prevented the user from accessing the Jaeger console. On startup the {JaegerName} Operator looks for the oauth-proxy image from the OpenShift cluster, and if not found it uses the image supplied in the arguments. The Operator has been updated to provide the missing arguments.
```

Comment:
Should we include this issue in the release notes? It was introduced and solved in this version. So, AFAIK, there are not published releases with that issue, right?

Reply:
Good point.  I'll remove it.